src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -451,72 +644,22 @@ private function fixNextName(Tokens $tokens, int $index, array &$uses, string $n
      * @param array{content: string, tokens: array<int>} $class
      * @param array<string, string>                      $uses
      */
-    private function shortenClassIfPossible(Tokens $tokens, array $class, array &$uses, string $namespaceName): void
+    private function shortenClassIfPossible(Tokens $tokens, array $class, array $uses, string $namespaceName): int
```

Comment:
Why do we need to return `int` here? Looking at `replaceClassWithShort()` it may be related to replacing symbol with short type which has fewer tokens, but I believe we should fill remaining slots with empty tokens (`new Token('')` or just use `Tokens::clearAt()`), so we don't change the tokens list during iterating through it. Later empty tokens would be removed.

Reply:
This is needed as `\` might be added for leading namespace because of `leading_backslash_in_global_namespace` option. Was quite pain to get it all right as the indexes are shifted because of it.

Reply:
Ah, OK, for global symbol and leading backslash we actually add new token so the tokens list grows. Did not think about this case. Maybe it would be good to add `@return` with proper description so it's clear for any John Doe in the future?

Reply:
I am fan of clean code, I think return comment will only add LoC and sooner or later the comments will become obsolete, ie. only misleding John Doe. But if you want, feel free to push anything into by branch.